<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberCtrlMaskWdg</class>
 <widget class="QWidget" name="MemberCtrlMaskWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MemberCtrlMaskWdg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0, 0, 0, 204);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>77</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="CMyButton" name="pushButton_mainView">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/main_view_click.png);
}

QPushButton:hover{
border-image: url(:/vhallActive/img/vhallactive/main_view_hover.png);
}

QPushButton{
border-image: url(:/vhallActive/img/vhallactive/main_view.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CMyButton" name="pushButton_setSpeaker">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/setspeaker_click.png);
}

QPushButton:hover{
border-image: url(:/vhallActive/img/vhallactive/setspeaker_hover.png);
}

QPushButton{
border-image: url(:/vhallActive/img/vhallactive/setspeaker.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CMyButton" name="pushButton_camera">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-image: url(:/vhallActive/img/vhallactive/camera_normal.png);
}
QPushButton:hover{
	border-image: url(:/vhallActive/img/vhallactive/camera_hover.png);
}
QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/camera_click.png);
}

	
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CMyButton" name="pushButton_mic">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/mic_click.png);
}

QPushButton:hover{
border-image: url(:/vhallActive/img/vhallactive/mic_hover.png);
}

QPushButton{
border-image: url(:/vhallActive/img/vhallactive/mic_normal.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_switch">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border-image: url(:/vhallActive/img/vhallactive/switch_doc.png);}QPushButton:hover{border-image: url(:/vhallActive/img/vhallactive/switch_doc_hover.png);}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CMyButton" name="pushButton_KickOutRoom">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-image: url(:/vhallActive/img/vhallactive/off_speak.png);
}
QPushButton:hover{
border-image: url(:/vhallActive/img/vhallactive/off_speak_hover.png);
}
QPushButton:pressed{
border-image: url(:/vhallActive/img/vhallactive/off_speak_click.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>77</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CMyButton</class>
   <extends>QPushButton</extends>
   <header>CMyButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../VhallLive/project/VhallLive.qrc"/>
  <include location="../../Project/MainUI.qrc"/>
 </resources>
 <connections/>
</ui>
